cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -41,10 +41,8 @@ spec:
               cpu: 100m
               memory: 100Mi
           env: 
-            - name: INFLUXDB_EXTERNAL_URL
-              value: /api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb:api/db/
```

Comment:
This part was added to make Grafana accessible via the api-server proxy. Production users should be removing this. I am skeptical on removing this, since it affects off-the-box experience. WDYT?